backend/api/serializers/computetask.py

```diff
@@ -85,8 +85,12 @@ def get_value(self, task_output):
                 model = Model.objects.get(key=output_asset.asset_key)
                 data.append(ModelSerializer(context=self.context, instance=model).data)
             elif output_asset.asset_kind == FunctionOutput.Kind.ASSET_PERFORMANCE:
-                task_key, metric_key = output_asset.asset_key.split("|")
-                perf = Performance.objects.get(compute_task__key=task_key, metric__key=metric_key)
+                compute_task_key, metric_key, identifier = output_asset.asset_key.split("|")
+                task_output = ComputeTaskOutput.objects.get(
+                    task=compute_task_key,
+                    identifier=identifier,
+                )
+                perf = Performance.objects.get(compute_task_output=task_output, metric__key=metric_key)
```

Comment:
This section is a nightmare for performance (execution time), both curent and new implementation. 
This is not a very constructive comment, because I don't have a good alternative to offer, and anyway, it would be out of scope of this PR. But maybe it's worth to keep it in mind and do some (manual) tests on a project with a lot of data and 10  metrics (you know the one I'm thinking about) 🙂

Reply:
Yes I agree... But I had in mind that @guilhem-barthes was planning on deleting these get_asset and get_value, so it might be the solution ahah :)

Reply:
This is the leftover GT task, but still considering why we don't generate a key instead of concatenating keys

Reply:
Also, do we still need the `metric_key`? wouldn't be `compute_task_key|identifier` unique?

Reply:
Mmmmh this is a good question. before, the metric key was used by the frontend to serialize the graph. But with the work we did on the front with Milan, the metric key completely disapear from the frontend now. So we might want to also delete it from the performance...